runtime/src/main/kotlin/konan/Annotations.kt

```diff
@@ -101,18 +71,26 @@ public annotation class FixmeReflection
  */
 public annotation class FixmeConcurrency
 
-public annotation class FixmeInline
+/**
+ * Need to be fixed because of header/impl notation
+ */
+public annotation class FixmeMultiplatform
 
 /**
- * Need to be fixed.
+ * Need to be fixed because of random support.
  */
-public annotation class Fixme
+public annotation class FixmeRandom
 
+/**
+ * Escape analysis annotations.
+ */
 public annotation class Escapes(val who: Int)
 
 public annotation class PointsTo(vararg val onWhom: Int)
 
 /**
- * Need to be fixed because of header/impl notation
+ * Top level variable or object is thread local, and so could be mutable.
```

Comment:
This behaviour can't be implemented efficiently if we separate workers and OS threads.
I suppose we should support main-thread-only objects instead.